sdk/typescript/src/cryptography/multisig.ts

```diff
@@ -49,6 +49,7 @@ export type MultiSig = {
 };
 
 export const MAX_SIGNER_IN_MULTISIG = 10;
+export const MIN_SIGNER_IN_MULTISIG = 2;
```

Comment:
Although it may seem strange, nothing previously prohibited the use of  a 1-of-1 multisig address.
We should be allow 1-of-1 multisigs here, and change `MIN_SIGNER_IN_MULTISIG` to 1

Reply:
returned 1-1 multisig support

Reply:
Tests fixed too